MediaBrowser.Model/Entities/MediaStream.cs

```diff
@@ -112,7 +107,18 @@ public string DisplayTitle
                         attributes.Add("Default");
                     }
 
-                    return string.Join(" ", attributes);
+                    if (!string.IsNullOrEmpty(Title))
+                    {
+                        Console.WriteLine(Language);
```

Comment:
Is the `Console.WriteLine` supposed to be here? Looks like you forgot to delete it

Reply:
Yeah it's just for debugging. I'll keep it until I finalize how I'm filtering the tags as mentioned in my conversation above.